doc/developer-guides/hld/hv-vt-d.rst

```diff
@@ -306,9 +306,6 @@ deinitialization:
 .. doxygenfunction:: init_iommu
    :project: Project ACRN
 
```

Comment:
I notice this function (init_iommu_vm0_domain) was moved to another file, but still exists.  Is it correct to remove it from the documentation?